docs/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,13 @@
+= Reveal.js converter for Asciidoctor
```

Comment:
The reveal.js project seems very deliberate about keeping the "r" lowercase. So I think this title should read "reveal.js".

Also, define the `navtitle` attribute on this page as follows.

```
:navtitle: Introduction
```

This will allow the UI to configure the breadcrumbs correctly for this page.

Reply:
You're right but the capitalization feels a bit weird when other words are capitalized:

#### reveal.js Plugins

#### reveal.js Converter for Asciidoctor

Reply:
@mojavelinux I've added `:navtitle: Introduction` but the breadcrumb stills show "reveal.js Converter for Asciidoctor": 

[`0897e6b` (#389)](https://github.com/asciidoctor/asciidoctor-reveal.js/pull/389/commits/0897e6b7b49b99b2d619383169532613421db996)


![breadcrumb](https://user-images.githubusercontent.com/333276/100769459-5b366a00-33fc-11eb-95af-233cb8b1b2a0.png)

Did I miss something?

Reply:
Yes. Here's the logic: https://github.com/asciidoctor/asciidoctor-docs-ui/blob/main/src/partials/breadcrumbs.hbs#L11

Reply:
> the capitalization feels a bit weird when other words are capitalized

It may seem weird, but reveal.js goes out of its way to not capitalize the project name on its site, so we should respect that. According to the English style guides, a proper noun which is not capitalized should never be capitalized.

Reply:
So it should be "Asciidoctor reveal.js Converter Documentation". Maybe we can remove the "Asciidoctor", what do you think?

Reply:
That's the correct title. You could also drop "Converter". The important part is to set the `navtitle` because I will update the UI to use it. Then, the doctitle can be whatever you want it to be. Though I think "Asciidoctor reveal.js Documentation" seems clear enough.

Reply:
Done!

![ok](https://user-images.githubusercontent.com/333276/101059656-d16ad600-358e-11eb-96e3-7f868378546c.png)